packages/debug/src/browser/view/debug-configuration-widget.tsx

```diff
@@ -63,14 +69,10 @@ export class DebugConfigurationWidget extends ReactWidget {
     protected setStepRef = (stepRef: DebugAction | null) => this.stepRef = stepRef || undefined;
 
     render(): React.ReactNode {
-        const { configurations, currentConfiguration } = this.manager;
-        const options = Array.from(configurations).map((configuration, index) =>
-            <option key={index} value={configuration.name}>{configuration.name}</option>
-        );
-        const currentOption = currentConfiguration && currentConfiguration.name || '__NO_CONF__';
+        const { options, currentValue } = this;
         return <React.Fragment>
             <DebugAction run={this.start} label='Start Debugging' iconClass='start' ref={this.setStepRef} />
-            <select className='debug-configuration' value={currentOption} onChange={this.setCurrentConfiguration}>
+            <select className='debug-configuration' value={currentValue} onChange={this.setCurrentConfiguration}>
```

Comment:
I would prefer a single `this.currentValue` over the destructuring indirection. For multiple usages a local variable might make sense. Destructuring breaks find references for me.